packages/side-nav/src/SideNavItem.tsx

```diff
@@ -61,6 +60,17 @@ const defaultStyle = css`
   }
 `;
 
+const typographyStyle = {
+  [14]: css`
+    font-size: 14px;
+    line-height: 1em;
+  `,
+  [16]: css`
+    font-size: 16px;
+    line-height: 1.5em;
```

Comment:
`em` units are proportional, so 1em should work in both cases here. Why do the sizes need different line height definitions?

Reply:
The pattern we've defined is 14px => 20px line height; 16px => 24px line height